lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -522,9 +521,13 @@ export class PublicClientApplication {
      * Use to log out the current user, and redirect the user to the postLogoutRedirectUri.
      * Default behaviour is to redirect the user to `window.location.href`.
      */
-    logout(): void {
+    logout(account: IAccount, authorityString?: string): void {
```

Comment:
This can instead request an object, similar to the other APIs? e.g. `LogoutRequest`

Reply:
Will tackle this in a future PR